src/scanoss/agent/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+#[=======================================================================[
+SPDX-License-Identifier: GPL-2.0-only
+SPDX-FileCopyrightText: © 2021 Avinal Kumar <avinal.xlvii@gmail.com>
+#]=======================================================================]
+set(FO_CWD ${CMAKE_CURRENT_SOURCE_DIR})
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${FO_C_FLAGS}")
+cmake_minimum_required(VERSION 3.23)
```

Comment:
Any reason to use `3.23` as min version instead of `3.13` defined everywhere else?

Reply:
Hi @GMishx, No particular resason, just the one installed on my System. Please, tell me if 3.13 is fine or it should be replacede with 3.19 to fit installarion requierements (https://github.com/fossology/fossology/wiki/Install-from-Source)